Libraries/Components/TextInput/TextInput.js

```diff
@@ -1191,6 +1191,7 @@ function InternalTextInput(props: Props): React.Node {
         accessibilityState={props.accessibilityState}
         nativeID={props.nativeID}
         testID={props.testID}
+        enableFocusRing={props.enableFocusRing}
```

Comment:
Can you add this comment on the line you're adding? `// TODO(macOS GH#774)`

It helps us keep track of purposeful diffs we make so version upgrades are much easier.